### ISSUE TITLE
smalcivilboat

### ISSUE THREAD

**Issue by WongIong** [ZombieStriker/QualityArmoryVehicles2#49]
- [x] The boat have to place at the waterside but not at the water surface
- [x]  Also it would be blocked by seagrass
- [ ] Once it get onto land, there is not way to push it back to water expect picking it up.

**Comment by Lorenzo0111** [comment]
Issue 1 and 2 have been fixed. What do you suggest for issue 3?

**Comment by Lorenzo0111** [comment]
should be fixed. open a new issue if it still doesn't work

**Comment by WongIong** [comment]
When will you release this update?

**Comment by Lorenzo0111** [comment]
Today

**Comment by WongIong** [comment]
Great

**Comment by WongIong** [comment]
Have you released it?
